docs/build-with-powerloom/use-cases/existing-implementations/eth_price_tracking.md

```diff
@@ -0,0 +1,149 @@
+- `POWERLOOM_REPORTING_URL` : The URL for reporting to Powerloom.
+
+#### Optional
+
+- `IPFS URL` : The URL for the IPFS (InterPlanetary File System) service in HTTPS(s) (e.g. `https://ipfs.infura.io:5001`) multiaddr format (e.g. `/dns/ipfs.infura.io/tcp/5001/https`)
+- `IPFS_API_KEY`: The API key for the IPFS service (if required).
+- `IPFS_API_SECRET`: The API secret for the IPFS service (if required).
+- `SLACK_REPORTING_URL`: The URL for reporting to Slack.
+- `WEB3_STORAGE_TOKEN`: The token for Web3 Storage. You can generate or retrieve this token from your [API token page](https://web3.storage/) after signing up for a free plan at web3.storage.
+
+### Snapshot Building
+
+:::info
+It is recommended to look at the docs on [snapshot generation](/docs/protocol/specifications/snapshotter/snapshot-build).
+:::
+
+Snapshotter node has a simple interface as part of `GenericProcessorSnapshot` class that business logic specific computes need to implement. The rest of the heavy lifting around reliable submissions and finalization is taken care of by the rest of the features of the node.
+
+In this case, `EthPriceProcessor` located in [`snapshotter/modules/computes`](https://github.com/PowerLoom/snapshotter-computes/blob/eth_price_monitor/eth_price_tracking.py) is the sole compute class for ETH price tracking. This class implements the [`GenericProcessorSnapshot`](https://github.com/Powerloom/pooler/blob/main/snapshotter/utils/callback_helpers.py) interface, most notably the `compute()` callback. This callback is where we build the snapshots.
+
+
+```python reference
+https://github.com/PowerLoom/snapshotter-computes/blob/eth_price_monitor/eth_price_tracking.py#L1-L45
+```
+
+`get_eth_price_usd()` is located in [`snapshotter/modules/computes`](https://github.com/PowerLoom/snapshotter-computes/blob/eth_price_monitor/eth_price_tracking.py) and is a utility function that calculates the price of ETH as a weighted average of its price conversion ratios against DAI, USDC and USDT in the corresponding Uniswap V2 pair contracts. The weights assigned are relative to the total liquidity locked against the W(rapped)ETH token across the three pair contracts.
+
+```python reference
+https://github.com/PowerLoom/pooler/blob/main/snapshotter/utils/snapshot_utils.py#L140-L154
+```
+
+### Set Up the Codebase
+
+Setup the codebase by running the `bootstrap.sh` command in the terminal. Developers may use different shells, such as `bash` or `zsh`. Adjust the command accordingly.
+
+For bash:
+
+```bash
+bash bootstrap.sh
+```
+or
+```zsh
+zsh bootstrap.sh
+```
+
+### Run the Snapshotter Node
+
+
+```bash
+bash build.sh
+```
+or
+```zsh
+zsh build.sh
+```
+
+Once your node is running, the [core API server](/docs/build-with-powerloom/snapshotter-node/core-api/) can be found on [http://localhost:8002](http://localhost:8002). It comes with an easy to use SwaggerUI to test out the endpoints to retrieve snapshotted data, or monitor the node status among other things.
```

Comment:
http://localhost:8002 this will just return error 404, we should point the users to /docs